scraper/src/mindtouch2zim/processor.py

```diff
@@ -604,6 +612,39 @@ def _fetch_favicon_from_illustration(self, illustration: BytesIO) -> BytesIO:
         )
         return favicon
 
+    def _add_indexing_item_to_zim(
+        self,
+        creator: Creator,
+        title: str,
+        content: str,
+        fname: str,
+        zimui_redirect: str,
+    ):
+        """Add a 'fake' item to the ZIM, with proper indexing data
+
+        This is mandatory for suggestions and fulltext search to work properly, since
+        we do not really have pages to search for.
+
+        This item is a very basic HTML which automatically redirect to proper Vue.JS URL
+        """
+
+        redirect_url = f"../index.html#/{zimui_redirect}"
+        html_content = (
+            f"<html><head><title>{title}</title>"
+            f'<meta http-equiv="refresh" content="0;URL=\'{redirect_url}\'" />'
+            f"</head><body></body></html>"
+        )
+
+        logger.debug(f"Adding {fname} to ZIM index")
+        add_item_for(
+            creator=creator,
+            title=title,
+            path="index/" + fname,
+            content=bytes(html_content, "utf-8"),
```

Comment:
```suggestion
            content=html_content.encode("utf-8"),
```

that seems to be our convention

Reply:
oups, forgot to fix this, will do it now